dashboard/test/models/user_test.rb

```diff
@@ -1969,6 +1970,22 @@ def mock_geocoder_result(result)
     assert student.reload.deleted?
   end
 
+  test 'undestroy restores recent dependents only' do
+    teacher = create :teacher
+    old_section = create :section, teacher: teacher
+    Timecop.freeze 1.hour.ago do
+      old_section.destroy!
```

Comment:
Not the cleanest that the section is being destroyed before it was created. But such is the nature of time travel.

Reply:
![image](https://user-images.githubusercontent.com/8405316/29942924-59edc4e4-8e4c-11e7-8876-cf2239226728.png)